src/components/data/LocalVariablesContent.js

```diff
@@ -18,13 +18,31 @@ import * as StateHistory from 'src/libs/state-history'
 import * as Utils from 'src/libs/utils'
 
 
+export const getDisplayedAttribute = arr => {
+  const { key, value, description } = _.assign(...arr, {})
+  return [key, value, description ?? '']
+}
+
+const DESCRIPTION_SUFFIX = '__DESCRIPTION__'
```

Comment:
Historically we've done this sort of UI feature overlaying trickery with prefixes, so if someone were to download the data directly from the API it would be obvious; would you consider prefixing instead of suffixing? (see `referenceData_`) Obviously, you'd then have to flip the `endsWith` type things to `startsWith` type things, but that should be pretty straightforward, I think?

Reply:
No worries - all done.